libraries/ui-library/src/components/six-sidebar-item-group/six-sidebar-item-group.scss

```diff
@@ -23,6 +23,8 @@
 
 .six-sidebar-details__header {
   display: flex;
+  overflow-wrap: anywhere;
+  align-items: center;
```

Comment:
Thanks for the fix :-). Looks good, but I would keep the default alignment. I look a bit strange if the icons are not aligned with the first line:

Center
![image](https://github.com/six-group/six-webcomponents/assets/1475511/5a018010-234f-4fcc-9e6e-03cd76c563a2)

Default
![image](https://github.com/six-group/six-webcomponents/assets/1475511/5c078a67-31ba-4273-8579-7a8b0fa7fc69)

Reply:
I wasn't too sure either, I agree that it looks clearer if the icon stays on the first line. Will adapt this tomorrow :)

Reply:
Removed the center alignment